cmd/terramate/cli/stacks_globals_test.go

```diff
@@ -125,6 +127,67 @@ stack "/stacks/stack-1":
 
 stack "/stacks/stack-2":
 	str = "string"
+`,
+			},
+		},
+		{
+			name: "three stacks only two has globals, wd = stack3",
+			layout: []string{
+				"s:stacks/stack-1",
+				"s:stacks/stack-2",
+				"s:stack3",
+			},
+			wd: "/stack3",
+			globals: []globalsBlock{
+				{
+					path: "/stacks/stack-1",
+					add: globals(
+						str("str", "string"),
+					),
+				},
+				{
+					path: "/stack3",
+					add: globals(
+						str("str", "stack3-string"),
+					),
+				},
+			},
+			want: runExpected{
+				Stdout: `
+stack "/stack3":
+	str = "stack3-string"
+`,
+			},
+		},
+		{
+			name: "three stacks with globals, wd = stacks",
```

Comment:
Maybe add a test case when wd has no stacks inside, like you have stacks with globals but wd is some internal dir that is empty (should be empty result etc)